src/batches/setBatch3Dto4D.m

```diff
@@ -0,0 +1,30 @@
+% (C) Copyright 2020 CPP BIDS SPM-pipeline developers
+
+function matlabbatch = setBatch3Dto4D(volumesList, outputName, dataType, RT)
+  %
+  % Set the batch for 3D to 4D conversion
+  %
+  % USAGE:: matlabbatch = setBatch3Dto4D(volumesList, outputName, dataType, RT)
+  %
+  %   matlabbatch = setBatch3Dto4D(volumesList, outputName, dataType, RT)
+  %
+  % :param volumesList: Obligatory argument. List of volumes to be converted in a single 4D brain
```

Comment:
```suggestion
  % :param volumesList: List of volumes to be converted in a single 4D brain
```
I think that we don't need to memtion if arguments are obligatory.

I would mention the optional ones in the `USAGE` section like this

```matlab
USAGE:: 

   foo(barObligatory1, barObligatory2 [, barOptional1])
```